setup.py

```diff
@@ -27,21 +37,22 @@ def package_files(directory):
 
 setup(
     name='robohive',
-    version='0.6.0',
+    version='0.7.0',
     license='Apache 2.0',
     packages=find_packages(),
-    package_data={"": extra_files},
+    package_data={"": extra_files+['../robohive_init.py']},
     include_package_data=True,
-    description='environments simulated in MuJoCo',
+    description='A Unified Framework for Robot Learning',
     long_description=read('README.md'),
     long_description_content_type="text/markdown",
     url='https://github.com/vikashplus/robohive.git',
-    author='Movement Control Lab, UW',
+    author='Vikash Kumar',
+    author_email="vikahsplus@gmail.com",
     install_requires=[
         'click',
         'gym==0.13',
```

Comment:
maybe you want to remove this default dependenvy on gym if gynasium is an alternative

Reply:
I want to keep the default the same for now as its more stable. I plan to replace it in the next release.

Reply:
ah, I thought about it and decided to be forward-facing. Accepting your suggestion and moving the install to point to gymnasium